### ISSUE TITLE
Cypress should set a mocha slow threshold for more reasonable reporting

### ISSUE THREAD

**Issue by paulfalgout** [cypress-io/cypress#447]
I realize that this is something I can set, and that test speeds will probably be all over the map depending on what I'm testing, but I assume right now that cypress is using the default slow threshold which prints that red (123ms) after a "slow" test.  Right now our non-cypress users on our team look at CI and say, "wow your tests are having some issues"

So I guess I'm wondering if there's a better threshold to set it to where it might provide meaning for the general case.. and if so, can that be the default in cypress?

**Comment by brian-mann** [comment]
Agreed. Mocha slow threshold is completely inappropriate for e2e/integration testing. We'll file this away as "expose mocha to be configurable" either via `cypress.json` or as code in a `support/index.js` file.

**Comment by brian-mann** [comment]
`-s, --slow <ms>                         "slow" test threshold in milliseconds [75]` is the option in Mocha.

Perhaps we should raise this to a default of `5000ms` and then expose something in `cypress.json` like `slowTestThreshold`.

It's probably possible to tell mocha about this right now in a root before / beforeEach hook with `this.slow(...)`

**Comment by brian-mann** [comment]
In addition, we need to lift off this property from runnables else `this.slow(...)` will not work.

**Comment by brian-mann** [comment]
This is finally about to be done for good ;-)

**Comment by Bkucera** [comment]
@jennifer-shehane @brian-mann  what's the timeline on this? I'd love to see this go into 4.0 if possible. Minimize the red!

![image](https://user-images.githubusercontent.com/14625260/50375660-27772680-05cf-11e9-9923-c0fdf19fef96.png)

**Comment by jennifer-shehane** [comment]
I feel like the first part of this was done, and the second part regarding increasing thresholds was never implemented. Would probably require looking at some averages we've collected and assigning a new threshold. 

I'm not completely sure what @brian-mann had planned.

**Comment by fr0** [comment]
> This is finally about to be done for good ;-)

Did this ever happen? Using `mochawesome` with cypress leaves a lot of red in the console, and confuses users into thinking that tests are failing.

**Comment by FredTreg** [comment]
Sorry I am a bit confused by this thread. Was something implemented in the end? Do I have a way (any way) to move the threshold up? All my tests are 20s or more (and that's perfectly normal) so would be great if I could.

**Comment by Bkucera** [comment]
@FredTreg unfortunately not, but this PR is in review https://github.com/cypress-io/cypress/pull/4066

**Comment by carlos-granados** [comment]
Any news on this? I can't believe that something so simple and useful is taking so long

**Comment by blazejpawlak** [comment]
Is this task part of any ongoing or merged PR? #4066 was just closed.

**Comment by cguldner** [comment]
Please implement this, would be really nice to have

**Comment by jennifer-shehane** [comment]
There is no work done for this - there was a [stale PR](https://github.com/cypress-io/cypress/pull/4066) with some work for this that is now closed due to inactivity. Feel free to an open PR.

**Comment by jbpallingayan** [comment]
up on this.

**Comment by Parakoos** [comment]
Until this is done properly, is there a workaround for now? I tried calling this.slow(999999) in the tests but it seem to be ignored...

**Comment by jbpallingayan** [comment]
any updates on this?
